### ISSUE TITLE
Error in the copywithin method in array  example

### ISSUE THREAD

**Issue by Manivardhan111** [mdn/content#22767]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin

### What specific section or headline is this issue about?

copywithin method in array

### What information was incorrect, unhelpful, or incomplete?

const array1 = ['a', 'b', 'c', 'd', 'e'];

// copy to index 0 the element at index 3
console.log(array1.copyWithin(0, 3, 4));
// expected output: Array ["d", "b", "c", "d", "e"]

// copy to index 1 all elements from index 3 to the end
console.log(array1.copyWithin(1, 3));
// expected output: Array ["d", "d", "e", "d", "e"]

### What did you expect to see?

The expected output for the (array1.copyWithin(1, 3)) would be ["a", "d", "e", "d", "e"]

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/array/copywithin`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/array/copywithin/index.md
* Last commit: https://github.com/mdn/content/commit/5976f713c6b5e76052538cb8862b667a68c21414
* Document last modified: 2022-11-12T07:45:33.000Z

</details>

**Comment by Josh-Cena** [comment]
Yep, Brian is right. In fact, this is a live example and you can directly run the code and see the result, so it's trivially provable that the current text is correct.
